test/system/autoload_test.rb

```diff
@@ -45,4 +55,14 @@ class AutoloadTest < ApplicationSystemTestCase
       assert_text "Namespace: Hello, from Turbo page"
     end
   end
+
+  def click_button(*)
+    begin
+      assert_css "[data-stimulus-autoloading]"
+    rescue
+      # does not eagerly load any Stimulus
+    end
+    assert_no_css "[data-stimulus-autoloading]"
```

Comment:
This assertion is intended to wait for the autoloading to _complete_.

If we think using `assert` calls would make the test suite assertion count fluctuate too much, there might be other ways to wait on the presence or absence of elements matching the selector.